web-server/v0.4/src/utils/parse.js

```diff
@@ -290,3 +283,38 @@ export const parseIterationData = results => {
     ports: ports,
   };
 };
+
+export const filterIterationData = (responseData, selectedConfig) => {
+  const responseDataCopy = [];
+  for (var response in responseData) {
+    responseDataCopy[response] = [];
+    responseDataCopy[response].columns = cloneDeep(responseData[response].columns);
+    responseDataCopy[response].iterations = cloneDeep(responseData[response].iterations);
+    responseDataCopy[response].resultName = cloneDeep(responseData[response].resultName);
+  }
+
+  for (var response in responseDataCopy) {
+    const responseIterations = responseDataCopy[response].iterations;
+    let selectedConfigLength = Object.keys(selectedConfig).length;
+    if (selectedConfigLength > 0) {
+      const filteredResponseData = [];
```

Comment:
Perhaps consider renaming to `filteredResponseIterationData`?

Reply:
I agree